src/mscorlib/shared/System/Byte.cs

```diff
@@ -12,7 +12,7 @@ namespace System
     [Serializable]
     [StructLayout(LayoutKind.Sequential)]
     [TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
-    public struct Byte : IComparable, IConvertible, IFormattable, IComparable<Byte>, IEquatable<Byte>
+    public struct Byte : IComparable, IConvertible, IFormattable, IComparable<Byte>, IEquatable<Byte>, ISpanFormattable
```

Comment:
@jkotas, are there any concerns with implementing additional internal interfaces on these primitive types?  Mostly wondering about AOT and whether this significantly impacts size and the like.

Reply:
It is fine on non-generic types. It is a fixed cost for them.

The common generic types is where this hurts - it gets multiplied for them.

Reply:
> It is fine on non-generic types.

Cool, thanks.